[NewProduct] Add Pop!_OS

StephenBrown2 opened the pull request:
This adds details for Pop!_OS, a Linux distro made by the hardware company System76.

[Pop!_OS](https://pop.system76.com) is a free and open-source Linux distribution based on Ubuntu.

Releases of Pop!_OS are the same as Ubuntu, with new releases every six months in April and October. Long term support releases are made every two years, in April of even-numbered years. Each non-LTS release is supported for three months after the release of the next version, and LTS releases are supported for five years.

StephenBrown2 committed:
Add Pop!_OS information

StephenBrown2 committed:
Remove 'support' key

StephenBrown2 committed:
Add links for all Pop!_OS releases

marcwrobel commented on products/pop_os.md:
```diff
@@ -0,0 +1,68 @@
+title: Pop!_OS
+category: os
+iconSlug: popos
+permalink: /pop_os
```
An alternative URL, `/popos`, could be added :

    alternative_urls:
    -  /popos

marcwrobel replied:
Done.

marcwrobel commented on products/pop_os.md:
```diff
@@ -0,0 +1,68 @@
+releaseDateColumn: true
+activeSupportColumn: false
+eolColumn: General Support
+releasePolicyLink: https://support.system76.com/articles/pop-os-development-approach
```
This page is not about the release policy of Pop!_OS. Do you know if there is a page on the Pop!_OS website that explains clearly what you put in the description (`Releases of Pop!_OS are the same as Ubuntu...`) ?

marcwrobel replied:
Link replaced with https://pop.system76.com/ for the time being. Will be replaced if we can find a page describing the release policy.

marcwrobel replied:
According to the [contribution guide](https://github.com/endoflife-date/endoflife.date/blob/master/CONTRIBUTING.md) this link should contain information about the release policy and schedule and is NOT the product URL. We have no such link for this product, so let's remove it for now (it is optional).

marcwrobel commented on products/pop_os.md:
```diff
@@ -0,0 +1,68 @@
+activeSupportColumn: false
+eolColumn: General Support
+releasePolicyLink: https://support.system76.com/articles/pop-os-development-approach
+changelogTemplate: |
```
`changelogTemplate` can be removed: you had to set the `link` for each release and it seems Pop!_OS does not have a stable link to release changelogs (as far as I can see they use blog posts).

marcwrobel replied:
Done.

marcwrobel commented on products/pop_os.md:
```diff
@@ -0,0 +1,68 @@
+-   distrowatch: popos
+    regex: '^Distribution Release: Pop!_OS (?P<version>\d+\.\d+\.?\d+)$'
+    template: "{{version}}"
+# purls:
```
@captn3m0 any idea what the purl can be for Pop!_OS ?

marcwrobel replied:
Purls comments have been removed for now.

marcwrobel committed:
[Pop!_OS] Add /popos alternate URL

marcwrobel committed:
[Pop!_OS] Reformat file and remove purls comments

marcwrobel committed:
[Pop!_OS] Remove changelogTemplate

Releases and new features are announced via blog posts on https://blog.system76.com/.

marcwrobel committed:
[Pop!_OS] Update releasePolicyLink

It seems release policy is not documented on Pop!_OS website. Using the Pop!_OS landing page instead.

marcwrobel committed:
[Pop!_OS] Remove releasePolicyLink

marcwrobel reviewed (approved):
LGTM, @usta I let you do the final review.

usta commented on products/pop_os.md:
```diff
@@ -0,0 +1,77 @@
+
+>[Pop!_OS](https://pop.system76.com) is a free and open-source Linux distribution based on Ubuntu.
+
+Releases of Pop!_OS are the same as Ubuntu, with new releases every six months in April and October. Long term support releases are made every two years, in April of even-numbered years. Each non-LTS release is supported for three months after the release of the next version, and LTS releases are supported for five years. However, after the release of Pop!_OS 22.04, System76 announced that they w ...
```
Just maybe this `Releases of Pop!_OS are the same` might be better to use `Release schedule of ....`

usta replied:
what is your opionion ? @marcwrobel

marcwrobel replied:
Yes, I think that would be better. I updated it following your recommendation.

marcwrobel committed:
[Pop!_OS] Update description
